docs/source/Support/bskReleaseNotes.rst

```diff
@@ -52,6 +52,8 @@ Version |release|
   feedback output message.
 - Resolved a crash, induced by uninitialized memory, in the Camera module. The crash was first seen on Ubuntu 22 with
   gcc 9.5
+- Created a :ref:`scanningInstrumentController`, similar to :ref:`simpleInstrumentController`, but which constatly checks if the attitude error
+  and angular rate error (optional) are wihtin the requirement limits and sends an imaging command to a :ref:`simpleInstrument`.
```

Comment:
typo `wihtin`, `constatly`. I like to use the "Code Spell Checker" extension on VSCode to check for misspellings. Should check the rest of your changes for spell checks, I won't look for any more